<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTabs</class>
 <widget class="QDialog" name="DialogTabs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tabs</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Tab spacing (for hardware tab characters)</string>
     </property>
     <property name="buddy">
      <cstring>editTabSpacing</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="editTabSpacing"/>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkEmulateTabs">
     <property name="text">
      <string>&amp;Emulate tabs</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="labelEmulatedTabSpacing">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Emulated tab &amp;spacing</string>
     </property>
     <property name="buddy">
      <cstring>editEmulatedTabSpacing</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="editEmulatedTabSpacing">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QCheckBox" name="checkUseTabsInPadding">
     <property name="text">
      <string>&amp;Use tab characters in padding and emulated tabs</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogTabs</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
